homeassistant/components/sanix/strings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To get the Serial number and the Token you just have to sign in to the Sanix Dashboard and open the Help -> System version page. https://sanix.bitcomplex.pl/",
+        "data": {
+          "serial_no": "Serial number",
+          "token": "[%key:common::config_flow::data::access_token%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "service_date": {
+        "name": "Service date"
+      },
+      "time": {
+        "name": "Last status update"
+      },
```

Comment:
Can you maybe extract this sensor into a follow up PR? I am not sure if this is a valid sensor, since HA itself also tracks when the last time we got a status update. I don't want to block this PR from merging, so if you still want to have this discussion, please open a new PR after this one gets merged so we can discuss this separately

Reply:
The measurement from the device should be sent every 2 hours. But in reality, it's less frequent. Maybe it would be better to change this label to "Last measurement"?

Reply:
Honestly, I don't know. I just know that there has been discussion about sensors like these before, and I can't remember the outcome or where to find the discussion. Hence I proposed to split it off :)

Reply:
Sure